### ISSUE TITLE
Warning: Static used in non-static inline function

### ISSUE THREAD

**Issue by mgaudet** [eclipse/openj9#136]
```
In file included from cfreader.c:44:0:
attrlookup.gperf:70:22: error: ‘attributeHash’ is static but used in inline function ‘lookupKnownAttribute’ which is not static [-Werror]
 MethodParameters, CFR_ATTRIBUTE_MethodParameters, CFR_ATTRIBUTE_MethodParameters
                      ^
```

Error caused by compiling with GCC 5.3, with warnings as errors enabled. 

There probably needs to be an epic open for tracking compiler modernization work.

**Comment by mgaudet** [comment]
Related to #26, though not the same root cause at all.

**Comment by mgaudet** [comment]
I suspect this just got fixed by #161
